playbooks/templates/.env.j2

```diff
@@ -74,7 +74,7 @@ SKYNET_PORTAL_API="https://{{ cluster_domain }}"
 # Administrator contact email.
 # Needed for communication regarding SSL certification.
 # TODO: Deprecated.
-EMAIL_ADDRESS="{{ skynetlabs_devs_email }}"
+EMAIL_ADDRESS="{{ support_email }}"
```

Comment:
We should split one email var to `ssl_support_email` and `git_checkout_email` vars, because portal operators could use 2 different emails for this.